packages/shared/tailwind.config.js

```diff
@@ -157,6 +157,7 @@ module.exports = {
       mobileXL: '500px',
       mobileXXL: '550px',
       tablet: '656px',
+      tabletL: '680px',
```

Comment:
Hmm, not sure if we're supposed to do this? cc. @dailydotdev/web-team 
Can we use sth custom rather then adding to guidelines for this specific guideline?

Otherwise I would be fine if we make the change at `tablet` and not the exact size.

Reply:
I'm fine with doing a custom breakpoint. After I get confirmation on slack that this is how it's supposed to work: https://dailydotdev.slack.com/archives/C061T644XV3/p1705584848244849 I'll take another look on this.

Reply:
Thank you! This is how u can do it with tailwind ex. `[@media(min-width:680px)]`

Reply:
I refactored to custom breakpoint and also changed the code a little bit, so that we have a little bit less changes to compare against. Let me know what you think.